opflexagent/test/test_gbp_ovs_agent.py

```diff
@@ -102,9 +102,18 @@ def start(self, interval=0):
                        return_value='00:00:00:00:00:01'),
             mock.patch('neutron.agent.common.ovs_lib.BaseOVS.get_bridges'),
             mock.patch('oslo_service.loopingcall.FixedIntervalLoopingCall',
-                       new=MockFixedIntervalLoopingCall),
+                new=MockFixedIntervalLoopingCall),
             mock.patch('opflexagent.gbp_agent.GBPOpflexAgent.'
-                       '_report_state')):
+                '_report_state')]
+
+        @contextmanager
+        def nested(*contexts):
```

Comment:
Same here.  We should define it as a function in some base test class then each test API throughout different test classes can just invoke it directly.